src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -23,9 +23,11 @@
 
 // no precompiled headers
 #include "ci/ciUtilities.hpp"
+#include "compiler/compiler_globals.hpp"
 #include "gc/shared/barrierSet.hpp"
 #include "gc/shared/cardTable.hpp"
 #include "gc/shared/collectedHeap.hpp"
+#include "jvmci/jvmci_globals.hpp"
```

Comment:
This line can be removed since compiler/compiler_globals.hpp already includes jvmci_globals.hpp, right?

Reply:
Yes, you are right. Will remove. Thanks a lot.